netshare/models/doppelganger_tf_model.py

```diff
@@ -236,7 +236,9 @@ def _generate(
             data_attribute_outputs = pickle.load(f)
 
         num_real_attribute = len(data_attribute_outputs)
-        num_real_samples = len(
+        num_real_samples = get_config(
```

Comment:
when we share the generative model between two machines (i.e. for sharing porpuses) we don't have `data_train_npz` in the generation side